packages/core/src/Page/index.js

```diff
@@ -127,6 +128,9 @@ class Page {
     const prefixedTitle = this.pageConfig.titlePrefix
       ? this.pageConfig.titlePrefix + (this.title ? TITLE_PREFIX_SEPARATOR + this.title : '')
       : this.title;
+    const prefixedSuffixedTitle = this.pageConfig.titleSuffix
+      ? (prefixedTitle ? prefixedTitle + TITLE_SUFFIX_SEPARATOR : '') + this.pageConfig.titleSuffix
+      : prefixedTitle;
```

Comment:
for discussion: the use of ternary statements here is fine but does make it slightly less readable. Personally I would have opted for a clearer style, perhaps via a typical if statement.

Reply:
Idm changing it to use if else statement. I think using ternary statement is more consistent as the logic is more of less the same as prefixedTitle but with a longer variable name instead.

Reply:
@EltonGohJH Thanks for contributing to MarkBind! :)

I think `if` clauses to do the necessary modifications here might be simpler as well. But feel free to keep the current style if it turns out this is cleaner.

> or should I change both

We try to keep (especially distinctively) unrelated changes elsewhere for collaboration, reviewing and tracing bugs / etc., but sometimes feature additions / enhancements can come with simplifications to existing implementations to make the whole easier to read. There isn't a clear cut guideline between doing these simplifications before or concurrently, which really depends on the change, but in this case the affected amount and complexity seems small enough.

Reply:
@ang-zeyu 
The ternary expression seems more clean in my opinion. I think we can keep the ternary expression? The if loop seems way too verbose.
```
    let prefixedTitle;
    if (this.pageConfig.titlePrefix) {
      if (this.title) {
        prefixedTitle = this.pageConfig.titlePrefix + TITLE_PREFIX_SEPARATOR + this.title;
      } else {
        prefixedTitle = this.pageConfig.titlePrefix;
      }
    } else {
      prefixedTitle = this.title;
    }
    let prefixedSuffixedTitle;
    if (this.pageConfig.titleSuffix) {
      if (prefixedTitle) {
        prefixedSuffixedTitle = prefixedTitle + TITLE_SUFFIX_SEPARATOR + this.pageConfig.titleSuffix;
      } else {
        prefixedSuffixedTitle = this.pageConfig.titleSuffix;
      }
    } else {
      prefixedSuffixedTitle = prefixedTitle;
    }
```

Reply:
@EltonGohJH 

was thinking something like this, keeping the best of both:
```
let title = this.title;
if (this.pageConfig.titlePrefix) {
  title = this.pageConfig.titlePrefix + (this.title ? TITLE_PREFIX_SEPARATOR + this.title : '')
}
if (this.pageConfig.titleSuffix) {
  ...
}
```